scribejava-apis/src/main/java/com/github/scribejava/apis/XeroApi20.java

```diff
@@ -0,0 +1,37 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.oauth2.clientauthentication.ClientAuthentication;
+import com.github.scribejava.core.oauth2.clientauthentication.RequestBodyAuthenticationScheme;
+
+/**
+ * Xero.com Api
+ */
+public class XeroApi20 extends DefaultApi20 {
+
+    protected XeroApi20() {
+    }
+
+    private static class InstanceHolder {
+        private static final XeroApi20 INSTANCE = new XeroApi20();
+    }
+
+    public static XeroApi20 instance() {
+        return InstanceHolder.INSTANCE;
+    }
+
+    @Override
+    public String getAccessTokenEndpoint() {
+        return "https://identity.xero.com/connect/token";
+    }
+
+    @Override
+    protected String getAuthorizationBaseUrl() {
+        return "https://login.xero.com/identity/connect/authorize";
+    }
+
+    @Override
+    public ClientAuthentication getClientAuthentication() {
+        return RequestBodyAuthenticationScheme.instance();
+    }
```

Comment:
There was no need to Override this method. Default method (and the preferred one) is working with xero as well.